demos/continuous_batching/README.md

```diff
@@ -31,6 +31,7 @@ Here, we will also define the LLM engine parameters inside the `graph.pbtxt`.
 
 Install python dependencies for the conversion script:
 ```bash
+export PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
```

Comment:
this is already included in requirements.txt

Reply:
It is not. Once you scroll down to the last paragraph of the demo, there is vllm benchmarking script tutorial. It requires to install requirements from vllm repo. It has no extra index, that is why it fails. I do put this export here because it is the same on main branch. We should have it unified.